libs/domains/clusters/feature/src/lib/cluster-action-toolbar/cluster-action-toolbar.tsx

```diff
@@ -140,6 +143,28 @@ function MenuOtherActions({
 
   const canDelete = clusterStatus.status && isDeleteAvailable(clusterStatus.status)
 
+  const mutationInstallationGuide = () =>
```

Comment:
It's not a mutation but more an `opentInstallationGuideModal` ?

Reply:
https://github.com/Qovery/console/pull/1117/commits/ef30054dcd443a13930cb27ceef88c72d2ba845b